internal/ingress/annotations/authreq/main_test.go

```diff
@@ -243,6 +243,70 @@ func TestCacheDurationAnnotations(t *testing.T) {
 	}
 }
 
+func TestKeepaliveAnnotations(t *testing.T) {
+	ing := buildIngress()
+
+	data := map[string]string{}
+	ing.SetAnnotations(data)
```

Comment:
nit: I would move this to inside the test loop, so you wont leak the map between tests executions

Reply:
that's why it's a nit :)